ci/do_ci.sh

```diff
@@ -5,11 +5,13 @@ set -e
 export BUILDIFIER_BIN="/usr/local/bin/buildifier"
 
 function do_build () {
+    setup_gcc_toolchain
     bazel build $BAZEL_BUILD_OPTIONS --verbose_failures=true //:nighthawk_client //:nighthawk_test_server \
-        //:nighthawk_service
+    //:nighthawk_service
 }
 
 function do_test() {
+    setup_gcc_toolchain
```

Comment:
FWIW, if we move to only clang we will be in a good place when it comes to both coverage and normal test/build; Envoy basically only uses gcc tests as a regression now, life is faster/better with Clang.

Reply:
cool, done